sqlalchemy_redshift/__init__.py

```diff
@@ -1,18 +1,29 @@
-from pkg_resources import get_distribution, parse_version
+from pkg_resources import DistributionNotFound, get_distribution, parse_version
 
 try:
     import psycopg2  # noqa: F401
-    if get_distribution('psycopg2').parsed_version < parse_version('2.5'):
-        raise ImportError('Minimum required version for psycopg2 is 2.5')
-        # requires sqlalchemy.sql.base.DialectKWArgs.dialect_options, new in
-        # version 0.9.2
 except ImportError:
     raise ImportError(
         'No module named psycopg2. Please install either '
         'psycopg2 or psycopg2-binary package for CPython '
         'or psycopg2cffi for Pypy.'
     )
 
+for package in ['psycopg2', 'psycopg2-binary', 'psycopg2cffi']:
+    try:
+        if get_distribution(package).parsed_version < parse_version('2.5'):
+            raise ImportError('Minimum required version for psycopg2 is 2.5')
+        break
+    except DistributionNotFound:
+        pass
+else:
+    raise ImportError(
+        'A module was found named psycopg2, '
+        'but the version of it could not be checked '
+        'as it was neither the Python package psycopg2, '
+        'psycopg2-binary or psycopg2cffi.'
```

Comment:
It seems unlikely, but if someone wants to import some unknown variant of psycopg2 besides these three, I'd prefer to pass here rather than raising an import error. Or perhaps best would be to log a warning message.

Reply:
Alright, I worked with the goal of extending the existing check to also cover the other two psycopg2 packages, and thought it would be good to still give an error in case any of the three packages weren't found.

I've removed the exception now so the version is only checked if any of the packages are found.

Reply:
> thought it would be good to still give an error in case any of the three packages weren't found

In my mind, that scenario should be covered by the `import psycopg2` statement; it that doesn't raise an error, then the user must have installed some package intended to provide psycopg2 functionality.

Thanks for making the change.